sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Drop-student-assessments-index-only-one-assessment-data-null.sql

```diff
@@ -0,0 +1 @@
+DROP INDEX IF EXISTS sims.student_assessments_only_one_assessment_data_null;
```

Comment:
We no longer use the `IF EXISTS` since the migrations will ensure the DB state and the item being removed should be present otherwise the DB is not in the expected state.

Reply:
Ok. I'll remove `IF EXISTS`.